src/xdg-autostart-generator/xdg-autostart-condition.c

```diff
@@ -0,0 +1,39 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include "main-func.h"
+#include "strv.h"
+
```

Comment:
maybe add a quick comment here saying that this is a condition check binary to be plugged into ExecCondition=. Might not be ovious otherwise to the sporadic reader